org.nanuc.Axolotl.yml

```diff
@@ -39,168 +29,74 @@ finish-args:
-    build-commands:
-      - install -Dm 644 electron-linux-arm64-20.2.0.zip ${BUNDLER_CACHE}/electron-linux-arm64-20.2.0.zip
-    sources:
-      - type: file
-        url: https://github.com/electron/electron/releases/download/v20.2.0/electron-v20.2.0-linux-arm64.zip
-        sha256: 1d2dd05f060786eae5ce687794aa071df8a5f5733b7b2efa1481de8db75d7738
-        dest-filename: electron-linux-arm64-20.2.0.zip
+      # Build axolotl-web
+      - npm --prefix=axolotl-web --offline run build
 
-  - name: axolotl-electron-bundle
-    buildsystem: simple
-    build-options:
-      env:
-        npm_config_offline: "true"
-        npm_config_cache: /run/build/axolotl-electron-bundle/flatpak-node/npm-cache
-        GOBIN: /app/bin
-        GO111MODULE: off
-        GOPATH: /run/build/axolotl-electron-bundle
-    subdir: src/github.com/nanu-c/axolotl
-    build-commands:
-      - make build-dependencies-axolotl-web
-      - make build-axolotl-web
-      # Specify bundler working directory (where Electron and Astilectron is)
-      - echo "$(jq '. + { working_directory_path:"/app/lib/astibundler" }' bundler.json)" > bundler.json
-      - make build-axolotl-electron-bundle &
-        while [ -e /proc/$! ]; do echo -n "."; sleep 100; done
-      - make install-axolotl-electron-bundle DESTDIR=${FLATPAK_DEST} INSTALL_PREFIX=/bin
-    sources:
-      - generated-axolotl-web-sources.0.json
-      - generated-axolotl-sources.json
-      - type: git
-        url: https://github.com/nanu-c/axolotl
-        tag: v1.6.0
-        commit: e521ab32b5bf315866a17f6844e5339abbc41e79
-        dest: src/github.com/nanu-c/axolotl
-        x-checker-data:
-          type: git
-          tag-pattern: "^v([\\d.]+)$"
+      # Build axolotl
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --features tauri --release --verbose
+      - install -Dm755 ./target/release/axolotl -t /app/bin/
 
-  - name: metadata
-    buildsystem: simple
-    subdir: src/github.com/nanu-c/axolotl
-    build-commands:
+      # Install metadata files
       - install -Dm 644 flatpak/org.nanuc.Axolotl.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
-      - install -Dm 644 flatpak/org.nanuc.Axolotl.appdata.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
-      - install -Dm 644 flatpak/web/org.nanuc.Axolotl.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 644 flatpak/org.nanuc.Axolotl.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm 644 flatpak/org.nanuc.Axolotl.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
     sources:
       - type: git
-        url: https://github.com/nanu-c/axolotl
-        tag: v1.6.0
-        commit: e521ab32b5bf315866a17f6844e5339abbc41e79
-        dest: src/github.com/nanu-c/axolotl
+        url: https://github.com/nanu-c/axolotl.git
+        #tag: v2.0.4
```

Comment:
Lets check out a tag here, not just a git hash. This is important I think, as it shows which version we are releasing.

Reply:
Sure, this is just a TODO to use a tag once it is available. `v2.0.4` is still missing a fix for Flatpak. That's the reason why I asked to tag a new release in Telegram.

Reply:
Got it! I only realised after my review that the PR is indeed marked as draft

Reply:
What are the outstanding issues re axolotl and Flatpak at the moment? Are there any issues?

Reply:
I am not aware of any. Flatpak should work equally well as the other packages.